src/sentry/processing/realtime_metrics/redis.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+from typing import Union
+
+from sentry.exceptions import InvalidConfiguration
+from sentry.utils import redis
+
+from . import base
+
+
+class RedisRealtimeMetricsStore(base.RealtimeMetricsStore):
+    """An implementation of RealtimeMetricsStore based on a Redis backend."""
+
+    def __init__(
+        self,
+        cluster: str,
+        counter_bucket_size: int,
+        counter_ttl: datetime.timedelta,
+    ) -> None:
+        """Creates a RedisRealtimeMetricsStore.
+
+        "cluster" is the name of the Redis cluster to use. "counter_bucket_size" is the size
+        in second of the buckets that timestamps will be sorted into when a project's counter is incremented.
+        "counter_ttl" is the duration that counter entries will be kept around for *after
+        the last increment call*.
+        """
+
+        self.cluster = redis.redis_clusters.get(cluster)
+        self._counter_bucket_size = counter_bucket_size
+        self._counter_ttl = int(counter_ttl / datetime.timedelta(milliseconds=1))
+        self._prefix = "symbolicate_event_low_priority"
+
+    def validate(self) -> None:
```

Comment:
what is this being used for? i can't find any usages of this anywhere; is this being magically invoked during initialization?

Reply:
Tbh I added it because the `Service` class in `sentry.utils.services` provides it. Might be superfluous.

Reply:
ahh, i see! that makes sense then, i'm tempted to remove this to see what it breaks but it would be better to leave this in so we don't break and/or at least fill in `Service`'s API.